scalableminds/webknossos
# Hybrid task with nml

- [x] blocked by performance problems of creating/duplicating volume tracings

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- create explorative annotation as baseAnnotation and copy the id
- create new task with the copied id as baseAnnotationId
- get task and see the baseAnnotation 

### Issues:
- fixes #4117 
- fixes #4187 

------
